### PR TITLE
integration/docker: add hot plug test

### LINKED ISSUE #675: add integration test for hot plugging
add and skip hot plugging test
https://github.com/clearcontainers/runtime/issues/789

### PR DESCRIPTION
**Pull request by devimc**
run a container with --device option and check if the
device is visible inside the container

fixes #675

Signed-off-by: Julio Montes <julio.montes@intel.com>

### CODE REVIEW

File: integration/docker/run_test.go [hunk 1]
```diff
@@ -97,3 +100,90 @@ var _ = Describe("run", func() {
...
+		Expect(ExistDockerContainer(id)).NotTo(BeTrue())
+		err = deleteLoopDevice(loopFile)
+		Expect(err).ToNot(HaveOccurred())
+		err = os.Remove(diskFile)
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	Context("hot plug a block device", func() {
+		It("should be attached", func() {
+			Skip("Issue: https://github.com/clearcontainers/runtime/issues/789")
```
Comment (jodh-intel) [inline integration/docker/run_test.go]: Nice.

File: integration/docker/run_test.go [hunk 2]
```diff
@@ -97,3 +100,90 @@ var _ = Describe("run", func() {
...
+// its loop device representation
+func createLoopDevice() (string, string, error) {
+	f, err := ioutil.TempFile("", "dd")
+	if err != nil {
+		return "", "", err
+	}
+	defer f.Close()
+
+	// create disk file
+	ddArgs := []string{"if=/dev/zero", fmt.Sprintf("of=%s", f.Name()), "count=1", "bs=5M"}
```
Comment (jodh-intel) [inline integration/docker/run_test.go]: ooi, why 5M?

Comment (devimc) [reply integration/docker/run_test.go]: I think 5M is enough for a disk file

**Comment by devimc** [comment]
branch_proxy: https://github.com/amshinde/clearcontainers-proxy/tree/vendoring-virtcontainers

**Comment by chavafg** [comment]
lgtm

**Pull request merged**